.github/workflows/release.yml

```diff
@@ -9,32 +10,29 @@ permissions:
   contents: write
 
 jobs:
-  goreleaser:
+  release:
     runs-on: ubuntu-latest
     steps:
       - name: Checkout
-        uses: actions/checkout@eef61447b9ff4aafe5dcd4e0bbf5d482be7e7871 # v4.2.1
+        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
         with:
           fetch-depth: 0
       - name: Setup Go
-        uses: actions/setup-go@0a12ed9d6a96ab950c8f026ed9f722fe0da7ef32 # v5.0.2
+        uses: actions/setup-go@0aaccfd150d50ccaeb58ebd88d36e91967a5f35b # v5.4.0
         with:
-          go-version-file: 'go.mod'
+          go-version-file: go.mod
       - name: golangci-lint
```

Comment:
I liked your suggestion [here](https://github.com/vmware/terraform-provider-nsxt/pull/1587#issuecomment-2766575146) to run the linter from `./.github/workflows/golangci-lint.yml` instead of having this duplication

Reply:
Yes, we can still do this - I was going to take a couple steps for this to be replicated across projects.